README.md

```diff
@@ -1,4 +1,4 @@
-# 3scale-operator
+# 3scale-operator [![CircleCI](https://circleci.com/gh/3scale/3scale-operator/tree/master.svg?style=svg)](https://circleci.com/gh/3scale/3scale-operator/tree/master)
```

Comment:
the badge next to the title? Maybe in the next line?

Reply:
I doesn't look that bad! :) 

<img width="482" alt="image" src="https://user-images.githubusercontent.com/2012568/54745739-8f3b0e80-4bca-11e9-99e0-b24e626f14e5.png">

But up to you, I can move it down one line.

Reply:
done!